config/routes.rb

```diff
@@ -6,7 +6,8 @@
 
   # Create named routes for each collection specified in tabs.yml
   YAML.load_file( File.join(Rails.root, "config", "repositories.yml") )["Catalog"]["repositories"].each do |coll|
-     get "#{coll[0]}" => "catalog#index", :search_field => "#{coll[1]['display']}", :repository => "#{coll[1]['display']}"
+     get "#{coll[1]['url']}" => "catalog#index", :search_field => "#{coll[1]['display']}", :repository => "#{coll[1]['display']}"
+
```

Comment:
@chrpr As mentioned above you'll need to remove the search_field and you can use the Repositories class to DRY it up (https://github.com/NYULibraries/findingaids/blob/development/lib/findingaids/repositories.rb):

```ruby
Findingaids::Repositories.repositories.each do |coll|
    get "#{coll.last['url']}" => "catalog#index", :repository => "#{coll.last['display']}"
end
```

If you feel like it you can also build out the Repositories class so that each repository in `.repositories` is a `Repository` and so you can query `Repositories.repositories.each {|repository| p repository.display}` etc.